src/removeStopwords.test.js

```diff
@@ -0,0 +1,22 @@
+import { removeStopwords } from './removeStopwords';
```

Comment:
better to create separate file for stopWords, constants.js responsibility is to hold code constants not localization constants.